repository/src/main/java/org/apache/atlas/repository/store/graph/v2/preprocessor/glossary/TermPreProcessor.java

```diff
@@ -120,12 +154,82 @@ private void processUpdateTerm(AtlasEntity entity, AtlasVertex vertex) throws At
             throw new AtlasBaseException(AtlasErrorCode.ACHOR_UPDATION_NOT_SUPPORTED);
         }
 
-        String vertexQnName = vertex.getProperty(QUALIFIED_NAME, String.class);
+        String vertexQName = vertex.getProperty(QUALIFIED_NAME, String.class);
+
+        entity.setAttribute(QUALIFIED_NAME, vertexQName);
+
+        String termGuid = GraphHelper.getGuid(vertex);
+        try {
```

Comment:
Confirm - Task should only be created in case term name is updated

Reply:
Yes, it will only fire up only in case of a term update.